docs/src/manual/solutions.md

```diff
@@ -113,8 +117,10 @@ Use [`raw_status`](@ref) to get a solver-specific string explaining why the
 optimization stopped:
 ```jldoctest solutions
 julia> raw_status(model)
-"kOptimal"
+"7"
 ```
+In this case, `7` is the `HiGHS.kHighsModelStatusOptimal` constant which isn't
+helpful unless you are familiar with the details of the HiGHS solver.
```

Comment:
Fixing this in https://github.com/jump-dev/HiGHS.jl/pull/102, so I might wait to merge this until HiGHS v1.1.1 is released.